lib/kontena/templates/haproxy/http_backends.text.erb

```diff
@@ -0,0 +1,42 @@
+<% services.each do |service| %>
+backend <%= service.name %>
+    option forwardfor
+    <% service.custom_settings.each do |setting| %>
+    <%= setting %>
+    <% end %>
+
+    <% if service.cookie %>
+    <% if service.cookie.empty? %>
```

Comment:
Styling: could there be some spacing to more easily identify the nesting ifs?

Reply:
I optimized this for output. Not sure how to make both output & code clean :/